in-depth/submission_guidelines.md

```diff
@@ -0,0 +1,177 @@
+
+For your image format, we'd ask that you pick whichever format best optimizes your image size vs its quality.  If your image has large blank or simplistic sections, a `.png` image will provide the best quality at a reasonable file size.  If your image is busy or has a lot of fine detail, we'd recommend you using a `.jpg` with a slightly reduced quality (80-95% should provide a good balance).  If you're unfamiliar with the differences between these formats, you can experiment to determine which option will provide the best balance of quality and file size.  If you have any questions, we are [here to help][GettingHelp].
+
+For thumbnail images, you should use a resolution of `270x220`.  The thumbnails are a locked size on the website, so providing a different resolution will result in your image being squashed.
+
+Fullsize image resolution isn't as strict.  These images will be proportionally scaled to the right size.  We do however recommend a resolution of at least `1440x900`, as these images can be viewed fullscreen by clicking on their magnifying glass icon.
+
+### Specifications
+
+* Match the filename of your `.md`
+* Choose an Optimal Format, `.png` or `.jpg`
+* Fullsize: `1440x900` or larger
+* Thumbnails: `270x220`
+
+## Metadata Breakdown
+
+### Heading
+
+The name of your theme or plugin.
+
+### Thumbnail
+
+The absolute location of your thumbnail image.
+
+### Categories
+
+These categories are displayed along the top of the page.  By clicking them, you can filter the results to just those of the category.  Each category below has a special short-name to use in your YAML.  We may add more categories in the future (let us know if you have any suggestions).
+
+#### Available Categories:
+
+##### Themes
+* Single Page (`single-page`) - A single page website, where all content is displayed on the `index` page.
+* Multiple Pages (`multi-page`) - A theme suitable for multiple pages, the opposite of single-page. (This category is kind of redundant isn't it.  It may not survive until the final version...)
+* Personal Website (`personal`) - A theme intended to be used as a portfolio of your work or to establish an online presence on your personal domain. (Maybe this would be better refered to as `portfolio`, as most websites you'd make yourself could be considered "personal")
+* Blog-Style (`blog`) - A theme styled with blogging or news posts in mind.
+
+##### Plugins
+* Administration (`admin`) - A plugin that adds administration capabilities to Pico.
+* Developer-Focused (`developer`) - A plugin that adds developer-focused functionality. (Not really a good category...)
+* Theming Extensions (`theming`) - A plugin that adds extends twig by adding extra variables or functionality.
+
+### Meta
+
+A short string of descriptive text that displays below your heading when you mouse over your item.
```

Comment:
I think I got "meta" from the class of the `<span>` it goes in.

At the time, a "description" variable was still being used for the *description*, not the markdown contents.

This sounds acceptable.  I think there was still code that would look at `item.description` if it existed. Did you remove that already?